runner/docker/README.md

```diff
@@ -34,3 +34,24 @@ To build a pipeline-specific container, you need to build the base container fir
    ```
 
    This command builds the `segment-anything-2` pipeline-specific container using the Dockerfile located at [docker/Dockerfile.segment_anything_2](docker/Dockerfile.segment_anything_2) and tags it as `livepeer/ai-runner:segment-anything-2`.
+
+4. **Set Up the `mediamtx` Container for Ingest**:
+
+```bash
+docker build -f docker/Dockerfile.mediamtx -t livepeer/ai-runner:mediamtx .
```

Comment:
This needs to be executed inside `docker/` dir (or you need to move `mediatx.yml` to another directory.
```suggestion
docker build -f Dockerfile.mediamtx -t livepeer/ai-runner:mediamtx .
```